src/pyforest/_importable.py

```diff
@@ -59,9 +59,9 @@ def __repr__(self, *args, **kwargs):
             # next line only adds imported_name into the local scope but does not trigger a new import
             # because the lazy_import was already imported via another trigger
             self.__maybe_import__()
-            return f"active pyforest.LazyImport of {eval(self.__imported_name__)}"
```

Comment:
I changed this no not run eval() as part of string construction. I assume it was a bug to do so.

Reply:
This was not a bug. We intended to let the user know the path of the package, that's why we used eval()